server/config/middleware.js

```diff
@@ -41,4 +44,6 @@ export default function middleware(app, express) {
   // pass the
   app.use(errorHandle.catchForward);
   app.use(errorHandle.handleError);
+
```

Comment:
This was for testing and can be deleted after review

Reply:
Sure thing.